app/src/main/java/com/discordtime/gpts/maps/view/MapViewFragment.kt

```diff
@@ -28,43 +36,33 @@ class MapViewFragment : Fragment(), OnMapReadyCallback {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+
         mMapView = view.findViewById(R.id.mapView) as MapView
         mMapView.onCreate(savedInstanceState)
         mMapView.onResume()
         mMapView.getMapAsync(this)
+        mapViewModel = ViewModelProviders.of(this)[MapViewModel::class.java]
```

Comment:
A question, when we're using Koin, should we use the "by inject" that it provides or we should use the ViewModelProviders from the Architeture Components.

Maybe if we use Koin we can start to inject what we need there on the ViewModel (e.g. IPlaceRepository).